capa/helpers.py

```diff
@@ -70,9 +70,21 @@ def assert_never(value) -> NoReturn:
     assert False, f"Unhandled value: {value} ({type(value).__name__})"  # noqa: B011
 
 
-def get_format_from_report(sample: Path) -> str:
-    report = json.load(sample.open(encoding="utf-8"))
+def load_json_from_path(json_path: Path):
+    import gzip
```

Comment:
thoughts on importing this at the top?

Reply:
yes, it would improve code readability. I'll do it